### PR TITLE
Add TYIpcDragSort component

### PR DESCRIPTION
**Pull request by HeLiangYu**
## Owner

- [x] Nickname(Tuya): 伽拉

## Description

tuya-panel-ipc-sdk add TYIpcDragSort component.

## Package

- [x] tuya-panel-ipc-sdk

## Type of change

Please select the relevant option.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by youngjuning — CHANGES_REQUESTED** [review]
@HeLiangYu 麻烦 commit 记录 rebase 一下

**Comment by HeLiangYu** [comment]
Add TYIpcDragSort Component

**Comment by HeLiangYu** [comment]
4ee38b153b84c4a7089890c26ace4ad8ef6373ca 
Add TYIpcDragSort Component

**Review by youngjuning — CHANGES_REQUESTED** [review]

File: packages/tuya-panel-ipc-sdk/src/ty-ipc-drag-sort/dragSort.tsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  Image,
+  GestureResponderEvent,
+  NativeScrollEvent,
+  NativeSyntheticEvent,
+} from 'react-native';
+import Res from './res';
+import styles from './style';
+import { TYIpcDragSortProps, orderItemInterface } from './interface';
+
+const DragSort: React.FC<TYIpcDragSortProps> & {
```
Comment (youngjuning) [inline packages/tuya-panel-ipc-sdk/src/ty-ipc-drag-sort/dragSort.tsx:18]: React.FC 被遗弃了已经，推荐使用 React.FunctionComponent

**Review by youngjuning — CHANGES_REQUESTED** [review]

File: packages/tuya-panel-ipc-sdk/src/ty-ipc-drag-sort/dragSort.tsx (hunk 1 above)
Comment (youngjuning) [reply packages/tuya-panel-ipc-sdk/src/ty-ipc-drag-sort/dragSort.tsx:18]: https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/

**Review by youngjuning — APPROVED** [review]
LGTM

**Pull request merged**